src/services/exit_order.py

```diff
@@ -0,0 +1,360 @@
+            -self._operator_stake_weight(operator_stats, self.total_predictable_validators_count),
+            -self._operator_predictable_validators(operator_stats),
+            self._validator_index(validator),
+        )
+
+    @staticmethod
+    def _operator_delayed_validators(operator_state: NodeOperatorPredictableState) -> int:
+        return operator_state.delayed_validators_count
+
+    @staticmethod
+    def _operator_targeted_validators_to_exit(operator_state: NodeOperatorPredictableState) -> int:
+        if operator_state.targeted_validators_limit_count is None:
+            return 0
+        return max(0, operator_state.predictable_validators_count - operator_state.targeted_validators_limit_count)
+
+    @staticmethod
+    def _operator_stake_weight(
+        operator_state: NodeOperatorPredictableState,
+        total_predictable_validators_count: int,
+    ) -> int:
+        stake_volume = 100 * operator_state.predictable_validators_count / total_predictable_validators_count
+        stake_volume_weight = operator_state.predictable_validators_total_age if stake_volume > 1 else 0
+        return stake_volume_weight
+
+    @staticmethod
+    def _operator_predictable_validators(operator_state: NodeOperatorPredictableState) -> int:
+        return operator_state.predictable_validators_count
+
+    @staticmethod
+    def _validator_index(validator: LidoValidator) -> int:
+        return int(validator.index)
+
+    def _decrease_node_operator_stats(self, validator: LidoValidator) -> None:
+        """
+        Sub particular validator stats from its node operator stats
+        We do it every time when validator is popped from the queue for resort the rest of queue
+        """
+        module_operator = self._no_index_by_validator(validator)
+        self.total_predictable_validators_count -= 1
+        self.lido_node_operator_stats[module_operator].predictable_validators_count -= 1
+        self.lido_node_operator_stats[module_operator].predictable_validators_total_age -= max(
+            0, self.blockstamp.ref_epoch - int(validator.validator.activation_epoch)
+        )
+
+    #  -- Internal methods to interact with operators state for sorting --
+    def _get_config(self):
+        orl = named_tuple_to_dataclass(
+            self.w3.lido_contracts.oracle_report_sanity_checker.functions.getOracleReportLimits().call(
+                block_identifier=self.blockstamp.block_hash,
+            ),
+            OracleReportLimits
+        )
+        validator_delayed_timeout_in_slots = Web3.to_int(
+            self.w3.lido_contracts.oracle_daemon_config.functions.get('VALIDATOR_DELAYED_TIMEOUT_IN_SLOTS').call(
+                block_identifier=self.blockstamp.block_hash,
+            )
+        )
+        self.v_conf = ValidatorToExitIteratorConfig(
+            orl.max_validator_exit_requests_per_report,
+            validator_delayed_timeout_in_slots
+        )
+
+    def _get_exitable_lido_validators(
```

Comment:
let's return a list instead of a generator

Reply:
will be fixed during refactoring